data/hai/hai reveal 5 philosophers.txt

```diff
@@ -850,7 +850,7 @@ mission "Hai Reveal [B09] Scanning Devil-Hide"
 			label moon
 			action
 				set "Hai Reveal [B09] Scanning Devil-Hide: label moon"
-			`	The moon? Hmmm... okay, it's just an ordinary chunk of rock, but radiation from the planet is disturbing the sensors. I can only see a few dozen meters below ground, even though the scanner should be able to look down at least ten times that far... Aha!"`
+			`	"The moon? Hmmm... okay, it's just an ordinary chunk of rock, but radiation from the planet is disturbing the sensors. I can only see a few dozen meters below ground, even though the scanner should be able to look down at least ten times that far... Aha!`
```

Comment:
Like 846 above could use its trailing quotes clipped the same as this one, actually. :)